apps/marketplace/components/Teams/Dashboard/TeamsOverview.js

```diff
@@ -30,6 +30,10 @@ export class TeamsOverview extends Component {
 
     return (
       <div className={styles.teams}>
+        <div className={styles.optin}>
```

Comment:
Did you try to use the margin class from `main.scss`?

Reply:
I did think about it but the component was already importing its own styles with margins etc so added to what was already there.